doonut-app-external-api/src/main/kotlin/com/doonutmate/member/service/strategy/StrategyMemberDelete.kt

```diff
@@ -0,0 +1,7 @@
+package com.doonutmate.member.service.strategy
+
+import com.doonutmate.member.controller.dto.DeleteRequest
+
+interface StrategyMemberDelete {
+    fun delete(req: DeleteRequest)
```

Comment:
분기문을 없애기 위한 supports 메서드가 필요할 것 같아요.

```suggestion
    fun supports(oauthType: OauthType): Boolean
    fun delete(req: DeleteRequest)
```